CHANGELOG.md

```diff
@@ -24,16 +24,26 @@ Adding a requirement of a major version of a dependency is breaking a contract.
 Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian/report/compare/release-3.11.4...master
+[Unreleased]: https://github.com/atlassian/report/compare/release-3.11.5...master
 
-### Fixed
-- Restore missing log label in failed regression test
+## [3.11.5] - 2022-10-20
+[3.11.5]: https://github.com/atlassian/report/compare/release-3.11.4...release-3.11.5
+
+### Changed
```

Comment:
I may be wrong, but doesn't "Changed" indicate breaking change, i.e. major version bump?

Reply:
I've checked other libraries changelogs and it seems we're using it this way.

However our reference https://keepachangelog.com/en/1.0.0/ shows ### Changed section is used for communicating non breaking changes too - which would make sense.

How would we communicate non breaking changes?

Reply:
It does. It _suggests_ that a feature (contract) was changed, ie. it's not compatible anymore. It's like `Removed` and `Added` at the same time. If no contract was changed, then it's just a transparent fix (`Fixed`).

If we don't think about SemVer here, then everything is `Changed`, a bugfix, a new API, removed API, deprecation, security, everything is some kind of change. We wouldn't have a use for those [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) headers.

Reply:
> However our reference https://keepachangelog.com/en/1.0.0/ shows ### Changed section is used for communicating non breaking changes too - which would make sense.

Good point. However, Keep a Changelog does not understand SemVer at all. They say:
> `this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).`

But they don't even follow [the first rule of SemVer](https://semver.org/spec/v2.0.0.html#spec-item-1), they didn't declare their API.

> Changed for changes in existing functionality. 

In SemVer context this suggests incompatibility. But ok, if we forgive KaC their SemVer shortcomings, then we can use `Changed` for non-API changes, but if an API changes, we need to say the old version is `Removed` and the new one is `Added`.

Reply:
I think that even if by industry `Changed` wouldn't be associated strictly to a breaking change it could be good to agree about it in our contribution circle. I can't see any other reasonable place where we can communicate if already contributed, yet not released, changes are breaking or not. Till now I was assuming the type of change is exactly for that.

We can have a situation where someone contributes code however doesn't do the release. We could minimize the effort for the releaser by giving an easy algorythm to resolve what is the next version number.